### PR TITLE
Adding content/pt/docs/reference/cheatsheet

### PR DESCRIPTION
**Pull request by femrtnz**
Adding content/pt/docs/reference/cheatsheet

### CODE REVIEW

File: content/pt/docs/reference/kubectl/cheatsheet.md [hunk 1]
```diff
@@ -0,0 +1,387 @@
...
+## Kubectl Autocomplete
+
+### BASH
+
+```bash
+source <(kubectl completion bash) # configuração de autocomplete no bash do shell atual, o pacote bash-completion precisa ter sido instalado primeiro.
+echo "source <(kubectl completion bash)" >> ~/.bashrc # para adicionar o autocomplete permanentemente no seu shell bash.
+```
+
+Você também pode usar um apelido abreviado para `kubectl` que também funciona com o autocomplete:
```
Comment (devlware) [inline content/pt/docs/reference/kubectl/cheatsheet.md]:
```suggestion
Você também pode usar uma abreviação para o atalho para `kubectl` que também funciona com o auto completar:
```

File: content/pt/docs/reference/kubectl/cheatsheet.md [hunk 2]
```diff
@@ -0,0 +1,387 @@
...
+```bash
+alias k=kubectl
+complete -F __start_kubectl k
+```
+
+### ZSH
+
+```bash
+source <(kubectl completion zsh)  # configuração para usar autocomplete no terminal zsh no shell atual
+echo "if [ $commands[kubectl] ]; then source <(kubectl completion zsh); fi" >> ~/.zshrc # adicionar autocomplete permanentemente para o seu shell zsh
```
Comment (devlware) [inline content/pt/docs/reference/kubectl/cheatsheet.md]:
```suggestion
echo "if [ $commands[kubectl] ]; then source <(kubectl completion zsh); fi" >> ~/.zshrc # adicionar auto completar permanentemente para o seu shell zsh
```

File: content/pt/docs/reference/kubectl/cheatsheet.md [hunk 3]
```diff
@@ -0,0 +1,387 @@
...
+```
+
+##  Contexto e configuração do Kubectl
+
+Defina com qual cluster do Kubernetes o `kubectl` se comunica e modifique os detales da configuração.
```
Comment (devlware) [inline content/pt/docs/reference/kubectl/cheatsheet.md]:
```suggestion
Defina com qual cluster Kubernetes o `kubectl` se comunica e modifique os detalhes da configuração.
```

File: content/pt/docs/reference/kubectl/cheatsheet.md [hunk 4]
```diff
@@ -0,0 +1,387 @@
...
+Veja a documentação [Autenticando entre clusters com o kubeconfig](/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) para
+informações detalhadas do arquivo de configuração.
+
+```bash
+kubectl config view # Mostrar configurações do kubeconfig mergeadas.
```
Comment (devlware) [inline content/pt/docs/reference/kubectl/cheatsheet.md]:
```suggestion
kubectl config view # Mostra configurações mescladas do kubeconfig.
```

File: content/pt/docs/reference/kubectl/cheatsheet.md [hunk 5]
```diff
@@ -0,0 +1,387 @@
...
+
+# defina um contexto utilizando um nome de usuário e o namespace.
+kubectl config set-context gce --user=cluster-admin --namespace=foo \
+  && kubectl config use-context gce
+ 
+kubectl config unset users.foo                       # excluir usuário foo
+```
+
+## Aplicar
+`apply` gerencia aplicativos através de arquivos que definem os recursos do Kubernetes. Ele cria e atualiza recursos em um cluster através da execução `kubectl apply`.
```
Comment (devlware) [inline content/pt/docs/reference/kubectl/cheatsheet.md]:
```suggestion
`apply` gerencia aplicações através de arquivos que definem os recursos do Kubernetes. Ele cria e atualiza recursos em um cluster através da execução `kubectl apply`.
```

File: content/pt/docs/reference/kubectl/cheatsheet.md [hunk 6]
```diff
@@ -0,0 +1,387 @@
...
+Esta é a maneira recomendada de gerenciar aplicativos Kubernetes em ambiente de produção. Veja a [documentação do Kubectl](https://kubectl.docs.kubernetes.io).
```
Comment (devlware) [inline content/pt/docs/reference/kubectl/cheatsheet.md]:
```suggestion
Esta é a maneira recomendada de gerenciar aplicações Kubernetes em ambiente de produção. Veja a [documentação do Kubectl](https://kubectl.docs.kubernetes.io).
```

File: content/pt/docs/reference/kubectl/cheatsheet.md [hunk 7]
```diff
@@ -0,0 +1,387 @@
...
+
+## Criando objetos
+
+Manifestos Kubernetes podem ser definidos em YAML ou JSON. A extensão do arquivo `.yaml`,
```
Comment (devlware) [inline content/pt/docs/reference/kubectl/cheatsheet.md]:
```suggestion
Manifestos Kubernetes podem ser definidos em YAML ou JSON. A extensão de arquivo `.yaml`,
```

File: content/pt/docs/reference/kubectl/cheatsheet.md [hunk 8]
```diff
@@ -0,0 +1,387 @@
...
+`.yml`, e `.json` pode ser usado.
+
+```bash
+kubectl apply -f ./my-manifest.yaml            # criar recurso (s)
+kubectl apply -f ./my1.yaml -f ./my2.yaml      # criar a partir de vários arquivos
+kubectl apply -f ./dir                         # criar recurso (s) em todos os arquivos de manifesto no diretório
+kubectl apply -f https://git.io/vPieo          # criar recurso (s) a partir de URL
+kubectl create deployment nginx --image=nginx  # iniciar uma única instância do nginx
+kubectl explain pods,svc                       # obtenha a documentação para pod e svc
```
Comment (devlware) [inline content/pt/docs/reference/kubectl/cheatsheet.md]: Can you please review this line, in the online doc it says:
"kubectl explain pods                           # get the documentation for pod manifests
"

File: content/pt/docs/reference/kubectl/cheatsheet.md [hunk 9]
```diff
@@ -0,0 +1,387 @@
...
+
+```
+
+## Visualizando e localizando recursos
+
+```bash
+# Obter comandos com saída simples
+kubectl get services                          # Listar todos os serviços do namespace
+kubectl get pods --all-namespaces             # Listar todos os pods em todos namespaces
+kubectl get pods -o wide                      # Listar todos os pods no atual namespace, com mais detalhes
```
Comment (devlware) [inline content/pt/docs/reference/kubectl/cheatsheet.md]:
```suggestion
kubectl get pods -o wide                      # Listar todos os pods no namespace atual, com mais detalhes
```

File: content/pt/docs/reference/kubectl/cheatsheet.md [hunk 10]
```diff
@@ -0,0 +1,387 @@
...
+kubectl get deployment my-dep                 # Listar uma implantação específica
+kubectl get pods                              # Listar todos os pods no namespace
+kubectl get pod my-pod -o yaml                # Obter o YAML de um pod
+kubectl get pod my-pod -o yaml --export       # Obtenha o YAML de um pod sem informações específicas do cluster
```
Comment (devlware) [inline content/pt/docs/reference/kubectl/cheatsheet.md]: Can you please review this line. Its not defined in the online doc.

File: content/pt/docs/reference/kubectl/cheatsheet.md [hunk 11]
```diff
@@ -0,0 +1,387 @@
...
+kubectl describe nodes my-node
+kubectl describe pods my-pod
+
+# Listar serviços classificados por nome
+kubectl get services --sort-by=.metadata.name
+
+# Listar pods classificados por contagem de reinicializações
+kubectl get pods --sort-by='.status.containerStatuses[0].restartCount'
+
+# Lista PersistentVolumes classificado por capacidade
```
Comment (devlware) [inline content/pt/docs/reference/kubectl/cheatsheet.md]:
```suggestion
# Listar PersistentVolumes classificado por capacidade
```

File: content/pt/docs/reference/kubectl/cheatsheet.md [hunk 12]
```diff
@@ -0,0 +1,387 @@
...
+kubectl get pv --sort-by=.spec.capacity.storage
+
+# Obtenha a versão da label de todos os pods com a label app=cassandra
```
Comment (devlware) [inline content/pt/docs/reference/kubectl/cheatsheet.md]:
```suggestion
# Obter a versão do rótulo de todos os pods com a rótulo app=cassandra
```

File: content/pt/docs/reference/kubectl/cheatsheet.md [hunk 13]
```diff
@@ -0,0 +1,387 @@
...
+kubectl get pods --selector=app=cassandra -o \
+  jsonpath='{.items[*].metadata.labels.version}'
+
+# Obter todos os nós dos workers  (use um seletor para excluir resultados que possuem um rótulo
```
Comment (devlware) [inline content/pt/docs/reference/kubectl/cheatsheet.md]:
```suggestion
# Obter todos os nós workers (use um seletor para excluir resultados que possuem um rótulo
```

File: content/pt/docs/reference/kubectl/cheatsheet.md [hunk 14]
```diff
@@ -0,0 +1,387 @@
...
+# nomeado 'node-role.kubernetes.io/master')
+kubectl get node --selector='!node-role.kubernetes.io/master'
+
+# Obter todos os pods em execução no espaço para nome
```
Comment (devlware) [inline content/pt/docs/reference/kubectl/cheatsheet.md]:
```suggestion
# Obter todos os pods em execução no namespace
```

File: content/pt/docs/reference/kubectl/cheatsheet.md [hunk 15]
```diff
@@ -0,0 +1,387 @@
...
+kubectl get pods --field-selector=status.phase=Running
+
+# Obter ExternalIPs de todos os nós
```
Comment (devlware) [inline content/pt/docs/reference/kubectl/cheatsheet.md]:
```suggestion
# Obter IPs externos de todos os nós
```

File: content/pt/docs/reference/kubectl/cheatsheet.md [hunk 16]
```diff
@@ -0,0 +1,387 @@
...
+kubectl get nodes -o jsonpath='{.items[*].status.addresses[?(@.type=="ExternalIP")].address}'
+
+# Listar nomes de pods pertencentes a um particular RC
```
Comment (devlware) [inline content/pt/docs/reference/kubectl/cheatsheet.md]:
```suggestion
# Listar nomes de pods pertencentes a um RC particular 
```

File: content/pt/docs/reference/kubectl/cheatsheet.md [hunk 17]
```diff
@@ -0,0 +1,387 @@
...
+# Mostrar marcadores para todos os pods(ou qualquer outro objeto Kubernetes que suporte rotulagem)
+kubectl get pods --show-labels
+
+# Verifique quais nós estão prontos
+JSONPATH='{range .items[*]}{@.metadata.name}:{range @.status.conditions[*]}{@.type}={@.status};{end}{end}' \
+ && kubectl get nodes -o jsonpath="$JSONPATH" | grep "Ready=True"
+
+# Listar todos os segredos atualmente em uso por um pod
+kubectl get pods -o json | jq '.items[].spec.containers[].env[]?.valueFrom.secretKeyRef.name' | grep -v null | sort | uniq
+
```
Comment (devlware) [inline content/pt/docs/reference/kubectl/cheatsheet.md]: Looks like there are some examples that are missing. Can you please check.

File: content/pt/docs/reference/kubectl/cheatsheet.md (hunk 4 above)
Comment (femrtnz) [reply content/pt/docs/reference/kubectl/cheatsheet.md]: Not sure we should use `mescladas` or `mergeadas`. Honestly I prefer the second one

File: content/pt/docs/reference/kubectl/cheatsheet.md (hunk 15 above)
Comment (femrtnz) [reply content/pt/docs/reference/kubectl/cheatsheet.md]: ExternalIPs is the type, so we should not translate.

File: content/pt/docs/reference/kubectl/cheatsheet.md (hunk 17 above)
Comment (femrtnz) [reply content/pt/docs/reference/kubectl/cheatsheet.md]: I didn't find any examples missing. Could you please explain?

File: content/pt/docs/reference/kubectl/cheatsheet.md (hunk 4 above)
Comment (devlware) [reply content/pt/docs/reference/kubectl/cheatsheet.md]: No problem. :)

File: content/pt/docs/reference/kubectl/cheatsheet.md (hunk 17 above)
Comment (devlware) [reply content/pt/docs/reference/kubectl/cheatsheet.md]: I checked the translation against https://kubernetes.io/docs/reference/kubectl/cheatsheet/. 
I believe the missing part is this:
```
# List all containerIDs of initContainer of all pods
# Helpful when cleaning up stopped containers, while avoiding removal of initContainers.
kubectl get pods --all-namespaces -o jsonpath='{range .items[*].status.initContainerStatuses[*]}{.containerID}{"\n"}{end}' | cut -d/ -f3
```

Comment (femrtnz) [reply content/pt/docs/reference/kubectl/cheatsheet.md]: Good catch @devlware ! just updated,  thanks

File: content/pt/docs/reference/kubectl/cheatsheet.md [hunk 18]
```diff
@@ -0,0 +1,387 @@
...
+kubectl exec my-pod -c my-container -- ls /         # Executar comando no pod existente (pod com vários contêineres)
+kubectl top pod POD_NAME --containers               # Mostrar métricas para um determinado pod e seus contêineres
+```
+
+## Interagindo com nós e cluster
+
+```bash
+kubectl cordon my-node                                                # Marcar meu nó como não agendável
+kubectl drain my-node                                                 # Drene o meu nó na preparação para manutenção
+kubectl uncordon my-node                                              # Marcar meu nó como agendável
```
[the remark this answers is not in the document]
Comment (femrtnz) [reply content/pt/docs/reference/kubectl/cheatsheet.md]: Cool. I think I will move to `Marcar o nó my-node como não agendável`

File: content/pt/docs/reference/kubectl/cheatsheet.md [hunk 19]
```diff
@@ -0,0 +1,387 @@
...
+kubectl exec my-pod -- ls /                         # Executar comando no pod existente (1 contêiner)
+kubectl exec my-pod -c my-container -- ls /         # Executar comando no pod existente (pod com vários contêineres)
+kubectl top pod POD_NAME --containers               # Mostrar métricas para um determinado pod e seus contêineres
+```
+
+## Interagindo com nós e cluster
+
+```bash
+kubectl cordon my-node                                                # Marcar meu nó como não agendável
+kubectl drain my-node                                                 # Drene o meu nó na preparação para manutenção
```
[the remark this answers is not in the document]
Comment (femrtnz) [reply content/pt/docs/reference/kubectl/cheatsheet.md]: I think I will move to `Drene o nó my-node na preparação para manutenção`

File: content/pt/docs/reference/kubectl/cheatsheet.md (hunk 12 above)
Comment (jhonmike) [reply content/pt/docs/reference/kubectl/cheatsheet.md]: `label` is better IHMO

File: content/pt/docs/reference/kubectl/cheatsheet.md [hunk 20]
```diff
@@ -0,0 +1,391 @@
...
+kubectl get pods --sort-by='.status.containerStatuses[0].restartCount'
+
+# Lista PersistentVolumes classificado por capacidade
+kubectl get pv --sort-by=.spec.capacity.storage
+
+# Obtenha a versão da label de todos os pods com a label app=cassandra
+kubectl get pods --selector=app=cassandra -o \
+  jsonpath='{.items[*].metadata.labels.version}'
+
+# Obter todos os nós workers (use um seletor para excluir resultados que possuem um rótulo
```
Comment (jhonmike) [inline content/pt/docs/reference/kubectl/cheatsheet.md]:
```suggestion
# Obter todos os nós workers (use um seletor para excluir resultados que possuem uma label
```
or use `rótulo` or `label`

File: content/pt/docs/reference/kubectl/cheatsheet.md [hunk 21]
```diff
@@ -0,0 +1,391 @@
...
+kubectl get pods --all-namespaces -o jsonpath='{range .items[*].status.initContainerStatuses[*]}{.containerID}{"\n"}{end}' | cut -d/ -f3
+
+# Listar eventos classificados por carimbo de data / hora
+kubectl get events --sort-by=.metadata.creationTimestamp
+
+# Compara o estado atual do cluster com o estado em que o cluster estaria se o manifesto fosse aplicado.
+kubectl diff -f ./my-manifest.yaml
+```
+
+## Atualizando recursos
```
Comment (jhonmike) [inline content/pt/docs/reference/kubectl/cheatsheet.md]:
```suggestion
## Atualizando Recursos
```

File: content/pt/docs/reference/kubectl/cheatsheet.md [hunk 22]
```diff
@@ -0,0 +1,391 @@
...
+ && kubectl get nodes -o jsonpath="$JSONPATH" | grep "Ready=True"
+
+# Listar todos os segredos atualmente em uso por um pod
+kubectl get pods -o json | jq '.items[].spec.containers[].env[]?.valueFrom.secretKeyRef.name' | grep -v null | sort | uniq
+
+# Listar todos os containerIDs de initContainer de todos os pods
+# Útil ao limpar contêineres parados, evitando a remoção de initContainers.
+kubectl get pods --all-namespaces -o jsonpath='{range .items[*].status.initContainerStatuses[*]}{.containerID}{"\n"}{end}' | cut -d/ -f3
+
+# Listar eventos classificados por carimbo de data / hora
```
Comment (jhonmike) [inline content/pt/docs/reference/kubectl/cheatsheet.md]: Original text: `# List Events sorted by timestamp`
```suggestion
# Listar eventos classificados por timestamp
```

File: content/pt/docs/reference/kubectl/cheatsheet.md [hunk 23]
```diff
@@ -0,0 +1,391 @@
...
+# Compara o estado atual do cluster com o estado em que o cluster estaria se o manifesto fosse aplicado.
+kubectl diff -f ./my-manifest.yaml
+```
+
+## Atualizando recursos
+
+A partir da versão 1.11 `rolling-update` foi descontinuado (veja [CHANGELOG-1.11.md](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.11.md)), utilize o comando `rollout` no lugar deste.
+
+```bash
+kubectl set image deployment/frontend www=image:v2               # Aplica o atualização contínua nos containers "www" do deployment "frontend", atualizando a imagem
```
Comment (jhonmike) [inline content/pt/docs/reference/kubectl/cheatsheet.md]: rollout vs autualização contínua? ...
sometimes you are using frontend and others front-end
```suggestion
kubectl set image deployment/frontend www=image:v2               # Aplica o rollout nos containers "www" do deployment "front-end", atualizando a imagem
```

File: content/pt/docs/reference/kubectl/cheatsheet.md [hunk 24]
```diff
@@ -0,0 +1,391 @@
...
+kubectl rollout history deployment/frontend                      # Verifica o histórico de implantações, incluindo a revisão
```
Comment (jhonmike) [inline content/pt/docs/reference/kubectl/cheatsheet.md]: sometimes you are using `deployment` and others `implantações`
```suggestion
kubectl rollout history deployment/frontend                      # Verifica o histórico do deployment, incluindo a revisão
```

File: content/pt/docs/reference/kubectl/cheatsheet.md [hunk 25]
```diff
@@ -0,0 +1,391 @@
...
+kubectl rollout undo deployment/frontend                         # Rollback para a implantação anterior
```
Comment (jhonmike) [inline content/pt/docs/reference/kubectl/cheatsheet.md]:
```suggestion
kubectl rollout undo deployment/frontend                         # Rollback para o deployment anterior
```

File: content/pt/docs/reference/kubectl/cheatsheet.md [hunk 26]
```diff
@@ -0,0 +1,391 @@
...
+kubectl rollout undo deployment/frontend --to-revision=2         # Rollback para uma revisão específica
+kubectl rollout status -w deployment/frontend                    # Acompanhe o status de atualização do "front-end" até sua conclusão sem interrupção 
+kubectl rollout restart deployment/frontend                      # Reinício contínuo da implantação "front-end"
```
Comment (jhonmike) [inline content/pt/docs/reference/kubectl/cheatsheet.md]: Reinício contínuo 🤔 
```suggestion
kubectl rollout restart deployment/frontend                      # Reinício contínuo do deployment "front-end"
```

File: content/pt/docs/reference/kubectl/cheatsheet.md [hunk 27]
```diff
@@ -0,0 +1,391 @@
...
+## Atualizando recursos
+
+A partir da versão 1.11 `rolling-update` foi descontinuado (veja [CHANGELOG-1.11.md](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.11.md)), utilize o comando `rollout` no lugar deste.
+
+```bash
+kubectl set image deployment/frontend www=image:v2               # Aplica o atualização contínua nos containers "www" do deployment "frontend", atualizando a imagem
+kubectl rollout history deployment/frontend                      # Verifica o histórico de implantações, incluindo a revisão
+kubectl rollout undo deployment/frontend                         # Rollback para a implantação anterior
+kubectl rollout undo deployment/frontend --to-revision=2         # Rollback para uma revisão específica
+kubectl rollout status -w deployment/frontend                    # Acompanhe o status de atualização do "front-end" até sua conclusão sem interrupção
```
Comment (jhonmike) [inline content/pt/docs/reference/kubectl/cheatsheet.md]: sometimes you are using frontend and others front-end
```suggestion
kubectl rollout status -w deployment/frontend                    # Acompanhe o status de atualização do "frontend" até sua conclusão sem interrupção 
```

**Comment by femrtnz** [comment]
Thanks a lot @jhonmike and @devlware. Please let me know if you see any other change

File: content/pt/docs/reference/kubectl/cheatsheet.md [hunk 28]
```diff
@@ -0,0 +1,391 @@
...
+`-o=custom-columns=<spec>` | Imprimir uma tabela usando uma lista separada por vírgula de colunas personalizadas
+`-o=custom-columns-file=<filename>` | Imprima uma tabela usando o modelo de colunas personalizadas no arquivo `<nome do arquivo>`
+`-o=json`     | Saída de um objeto de API formatado em JSON
+`-o=jsonpath=<template>` | Imprima os campos definidos em uma expressão [jsonpath](/docs/reference/kubectl/jsonpath) 
+`-o=jsonpath-file=<filename>` | Imprima os campos definidos pela expressão [jsonpath](/docs/reference/kubectl/jsonpath) no arquivo `<nome do arquivo>`
+`-o=name`     | Imprima apenas o nome do recurso e nada mais
+`-o=wide`     | Saída no formato de texto sem formatação com qualquer informação adicional e, para pods, o nome do nó está incluído
+`-o=yaml`     | Saída de um objeto de API formatado em YAML
+
+### Verbosidade da saída do Kubectl e debugging
```
Comment (jhonmike) [inline content/pt/docs/reference/kubectl/cheatsheet.md]:
```suggestion
### Verbosidade da Saída do Kubectl e Debugging
```

File: content/pt/docs/reference/kubectl/cheatsheet.md (hunk 8 above)
Comment (jhonmike) [reply content/pt/docs/reference/kubectl/cheatsheet.md]: up

File: content/pt/docs/reference/kubectl/cheatsheet.md (hunk 9 above)
Comment (jhonmike) [reply content/pt/docs/reference/kubectl/cheatsheet.md]: up

File: content/pt/docs/reference/kubectl/cheatsheet.md [hunk 29]
```diff
@@ -0,0 +1,391 @@
...
+```
+
+## Visualizando e localizando recursos
+
+```bash
+# Obter comandos com saída simples
+kubectl get services                          # Listar todos os serviços do namespace
+kubectl get pods --all-namespaces             # Listar todos os pods em todos namespaces
+kubectl get pods -o wide                      # Listar todos os pods no atual namespace, com mais detalhes
+kubectl get deployment my-dep                 # Listar uma implantação específica
```
[the remark this answers is not in the document]
Comment (jhonmike) [reply content/pt/docs/reference/kubectl/cheatsheet.md]: up

File: content/pt/docs/reference/kubectl/cheatsheet.md (hunk 11 above)
Comment (jhonmike) [reply content/pt/docs/reference/kubectl/cheatsheet.md]: up

**Comment by jhonmike** [comment]
still has unverified comments/suggestions

File: content/pt/docs/reference/kubectl/cheatsheet.md (hunk 10 above)
Comment (femrtnz) [reply content/pt/docs/reference/kubectl/cheatsheet.md]: Line removed

File: content/pt/docs/reference/kubectl/cheatsheet.md (hunk 12 above)
Comment (femrtnz) [reply content/pt/docs/reference/kubectl/cheatsheet.md]: Agree

File: content/pt/docs/reference/kubectl/cheatsheet.md [hunk 30]
```diff
@@ -0,0 +1,387 @@
...
+kubectl -n my-ns delete pod,svc --all                                     # Exclua todos os pods e serviços no namespace my-ns,
+# Excluir todos os pods que correspondem ao awk pattern1 ou pattern2
+kubectl get pods  -n mynamespace --no-headers=true | awk '/pattern1|pattern2/{print $1}' | xargs  kubectl delete -n mynamespace pod
+```
+
+## Interagindo com a execução de Pods
+
+```bash
+kubectl logs my-pod                                 # despejar logs de pod (stdout)
+kubectl logs -l name=myLabel                        # despejar logs de pod, com nome da label = myLabel (stdout)
```
[the remark this answers is not in the document]
Comment (femrtnz) [reply content/pt/docs/reference/kubectl/cheatsheet.md]: Concordo, `label` faz mais sentido aqui

**Comment by femrtnz** [comment]
> still has unverified comments/suggestions

Done finally :)

**Comment by VineethReddy02** [comment]
/hold
pending v1.18 Kubernetes release

**Comment by VineethReddy02** [comment]
/hold cancel
v1.18 release is out.

**Pull request merged**